features/tailormade_auth/src/main/java/com/future/tailormade_auth/feature/signUp/view/SignUpFragment.kt

```diff
@@ -41,29 +42,30 @@ class SignUpFragment : BaseFragment() {
 
       buttonSubmitForm.setOnClickListener {
         submitForm(
-          editTextNameSignUp.text.toString(),
-          editTextEmailSignUp.text.toString(),
-          editTextBirthDateSignUp.text.toString(),
-          editTextPasswordSignUp.text.toString(),
-          editTextConfirmPasswordSignUp.text.toString(),
+            editTextNameSignUp.text.toString(),
+            editTextEmailSignUp.text.toString(),
+            editTextBirthDateSignUp.text.toString(),
+            editTextPasswordSignUp.text.toString(),
+            editTextConfirmPasswordSignUp.text.toString(),
         )
       }
 
       buttonGoToSignIn.setOnClickListener {
         findNavController().navigate(
-          SignUpFragmentDirections.actionSignUpFragmentToSignInFragment()
-        )
+            SignUpFragmentDirections.actionSignUpFragmentToSignInFragment())
       }
     }
 
     return binding.root
   }
 
-  private fun isFormValid(
-    name: String, email: String, birthDate: String,
-    password: String, confirmPassword: String
-  ): Boolean =
-    name.isNotBlank() && email.isNotBlank() && email.isEmailValid() && birthDate.isNotBlank() && password.isNotBlank() && password.length >= 8 && confirmPassword.isNotBlank() && confirmPassword == password
+  override fun onActivityCreated(savedInstanceState: Bundle?) {
+    super.onActivityCreated(savedInstanceState)
+    showToolbar()
+  }
+
+  private fun isFormValid(name: String, email: String, birthDate: String,
+      password: String, confirmPassword: String): Boolean = name.isNotBlank() && email.isNotBlank() && email.isEmailValid() && birthDate.isNotBlank() && password.isNotBlank() && password.length >= 8 && confirmPassword.isNotBlank() && confirmPassword == password
```

Comment:
please add enter manually when code is expanded more than 200 char horizontally for better readability